aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -1,34 +1,35 @@
 ---
 remote_endpoint: ~
-name: "v1.10"
+name: "v1.10-plus"
 proposals:
   - name: step_1_increase_max_txn_gas
     metadata:
       title: "Increase max txn gas temporarily for framework upgrade"
       description: "Increase max txn gas temporarily for framework upgrade"
     execution_mode: MultiStep
     update_sequence:
-      - DefaultGasWithOverride:
+      - DefaultGasWithOverrideOld: # This is translated to `gas_schedule::set_gas_schedule(...)`.
```

Comment:
if randomness is release in an official release 1.x, then the release.yaml may have to be a bit special^
cc @sherry-x @perryjrandall @igor-aptos

Reply:
if randomness is not yet enabled, does the new DefaultGasWithOverride work, or is it required to use DefaultGasWithOverrideOld ? 

I understand that DefaultGasWithOverrideOld will not work after randomness is enabled, but I am asking for the opposite.

Reply:
the new `DefaultGasWithOverride` works as long as the new framework is in action. It's designed to work no matter randomness is on or off.